answerking_app/models/serializers.py

```diff
@@ -298,17 +305,95 @@ def validate_lineItems(self, line_items_data):
 
     class Meta:
         model = Order
-        fields = (
+        fields = [
             "id",
             "createdOn",
             "lastUpdated",
             "orderStatus",
             "orderTotal",
             "lineItems",
-        )
+            "owner",
+        ]
         depth = 3
 
 
+class ManagerAuthSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(
+        required=True,
+        validators=[UniqueValidator(queryset=User.objects.all())],
+        max_length=50,
+        min_length=4,
+    )
+    email = serializers.EmailField(
+        required=True,
+        validators=[UniqueValidator(queryset=User.objects.all())],
+    )
+    password = serializers.CharField(
+        write_only=True, required=True, validators=[validate_password]
+    )
+    password2 = serializers.CharField(
+        write_only=True,
+        required=True,
+    )
+
+    class Meta:
+        model = User
+        fields = (
+            "username",
+            "password",
+            "password2",
```

Comment:
1 single password should be sent, password confirmation should be done by the front end